tracer/transport.go

```diff
@@ -32,7 +32,7 @@ func newHTTPTransport(url string) *httpTransport {
 	}
 }
 
-func (t *httpTransport) Send(spans []*Span) (*http.Response, error) {
+func (t *httpTransport) Send(traces [][]*Span) (*http.Response, error) {
```

Comment:
Now we should send a list of traces instead of a list of spans like defined in the ``datadog-trace-agent`` API ``v0.3``.